Matlab.tmbundle/Syntaxes/MATLAB.tmLanguage

```diff
@@ -223,7 +227,7 @@
 					<key>begin</key>
 					<string>(?&lt;=\))[^\S\n]*(\()?</string>
 					<key>end</key>
-					<string>(\))?[^\S\n]*(?=;|(?&lt;=[^.])\n|%)</string>
+					<string>(\))?[^\S\n]*(?=;|(?&lt;!(?:\.\.\..*))\n|%)</string>
```

Comment:
I'm not sure about using `.*` within the lookback operator. Regex101 complains about it (*"a quantifier inside a lookbehind makes it non-fixed width"*), but it seems that oniguruma is able to handle it.